crypto/keys/secp256k1/internal/secp256k1/scalar_mult_cgo.go

```diff
@@ -5,7 +5,7 @@
 //go:build !gofuzz && cgo
 // +build !gofuzz,cgo
 
-//nolint // this nolint lets us use this file in its original and unmodified form.
+// nolint // this nolint lets us use this file in its original and unmodified form.
```

Comment:
please revert all changes to this directory. This is a fork, we shouldn't touch it

Reply:
ahhhh ok.  I will add it to the not touches in the makefile.